README.md

```diff
@@ -0,0 +1,118 @@
+# PyVista Tutorial
+
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/pyvista/pyvista-tutorial/HEAD)
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+
+## Abstract
+
+
+[PyVista](https://github.com/pyvista/pyvista) is a general purpose 3D visualization library used for over 500+ open source projects and many closed source projects for the visualization of everything from [computer aided engineering and geophysics to volcanoes and digital artwork](https://dev.pyvista.org/getting-started/external_examples.html).
+
+PyVista leverages [VTK](http://www.vtk.org) to provide a python-friendly API that is immediately usable without any prior knowledge of VTK and is being built as the 3D equivalent of [matplotlib](https://matplotlib.org/), with plugins to [Jupyter](https://jupyter.org/) to enable visualization of results using both server and client-side rendering.
+
+We will provide a hands-on tutorial accessible to anyone with internet access and a computer via open and accessible [example Jupyter notebooks](https://docs.pyvista.org/examples/index.html) through a comprehensive tutorial highlighting popular use cases.
+
+
+
+## Keywords
+
+[visualization](https://github.com/topics/visualization)
+[meshviewer](https://github.com/topics/meshviewer)
+[vtk](https://github.com/topics/vtk)
+[3d](https://github.com/topics/3d)
+
+## Other Information and Files
+See our examples at [PyVista Examples](http://docs.pyvista.org/examples/index.html)
+
+### Tutorial Description.
+
+- Use PyVista effectively to create 3D visualizations from a variety of datasets.
+- Understand the basic class and data structure of PyVista.
+- Be familiar of the various filters and features of PyVista.
+- Know which Python libraries which are used and can be used by PyVista.
+We see this tutorial catering to anyone who wants to visualize data in any domain, and this ranges from basic Python users to advanced power users.
+
+1. Basic knowledge of Python to get started. Be able to install jupyterlab on your machine and be up and running.
+2. Intermediate users will want to be familiar with [NumPy](https://numpy.org/) and other libraries that are compatible with PyVista, like `trimesh` or `meshio`.
```

Comment:
Let's make sure to link to trimesh and meshio

Reply:
```suggestion
2. Intermediate users will want to be familiar with [NumPy](https://numpy.org/) and other libraries that are compatible with PyVista, like [trimesh](https://trimsh.org/examples.html) or [meshio](https://github.com/nschloe/meshio).
```